setup.py

```diff
@@ -9,29 +9,21 @@
 # See LICENSE for more details.
 #
 
-from __future__ import print_function
-
 import glob
 import os
 import platform
 import sys
-from distutils.command.build import build as _build
-from distutils.command.clean import clean as _clean
-from distutils.command.install_data import install_data as _install_data
-from shutil import rmtree
+from distutils.command.build import build as _build  # TODO: deprecated in PY3.12
```

Comment:
Please leave don't include TODO comments, this will be dealt with when the time comes